solidus_rails/solidus_rails.gemspec

```diff
@@ -0,0 +1,26 @@
+# encoding: UTF-8
+require_relative '../core/lib/spree/core/version.rb'
+
+Gem::Specification.new do |s|
+  s.platform    = Gem::Platform::RUBY
+  s.name        = 'solidus_rails'
+  s.version     = Spree.solidus_version
+  s.summary     = 'Provides the rails glue for the legacy solidus frontend/backend/api'
```

Comment:
Do we want to call these `legacy` just yet? :)

Reply:
Updated